sql/5_0_2-to-5_0_3_upgrade.sql

```diff
@@ -446,12 +446,32 @@ ALTER TABLE `codes` MODIFY `code_text_short` varchar(255) NOT NULL default '';
 
 #IfNotColumnTypeDefault amendments created_time timestamp NULL
 ALTER TABLE `amendments` MODIFY `created_time` timestamp NULL COMMENT 'created time';
+SET @currentSQLMode = SELECT @@sql_mode;
```

Comment:
need parenthesis:
```
SET @currentSQLMode = (SELECT @@sql_mode);
```

Reply:
Also need to do this for the other `SET @currentSQLMode =`... statements below

Reply:
Fixed!